beacon_node/network/src/sync/backfill_sync/mod.rs

```diff
@@ -733,19 +735,10 @@ impl<T: BeaconChainTypes> BackFillSync<T> {
                     // Batch is not ready, nothing to process
                 }
                 BatchState::Poisoned => unreachable!("Poisoned batch"),
-                BatchState::Failed | BatchState::AwaitingDownload | BatchState::Processing(_) => {
-                    // these are all inconsistent states:
-                    // - Failed -> non recoverable batch. Chain should have been removed
-                    // - AwaitingDownload -> A recoverable failed batch should have been
-                    //   re-requested.
-                    // - Processing -> `self.current_processing_batch` is None
-                    return self
-                        .fail_sync(BackFillError::InvalidSyncState(String::from(
-                            "Invalid expected batch state",
-                        )))
-                        .map(|_| ProcessResult::Successful);
-                }
-                BatchState::AwaitingValidation(_) => {
+                BatchState::Failed
+                | BatchState::AwaitingDownload
+                | BatchState::Processing(_)
+                | BatchState::AwaitingValidation(_) => {
```

Comment:
Why are these grouped now?

Reply:
damn I think this got deleted in some bad merge. Thanks!